app/views/posts/index.html.erb

```diff
@@ -1,2 +1,35 @@
-<h1>Here your posts list with user_id </h1>
-<p>Find me in app/views/posts/index.html.erb</p>
+<div class="container">
+<div class="user_profile">
+<div class="photos">
+    <ul class="photo"> 
+   <li> <img src="<%= @user.photo %>"> </li>
+   </ul>
+</div>
+<div class="names_posts">
+    <ul class="name_post"> 
+   <li class="list_name"> <%= @user.name %> </li>
+   <li class="list_countpost">Number of post: <%= @user.postscounter %> </li>
+   </ul>
+</div>
+ </div>
+
+ <div class="post">
+    <% @user.posts.each do |post| %>
+    <ul class="post_info width"> 
+    <h3 class="title"><%= post.title%> #<%= post.id%></h3>
+   <li class="bio_content"> <%= post.text %> </li>
+   <li class="list_countcomentlikes">Comments: <%= post.commentscounter %>, Likes: <%= post.likescounter %> </li>
+   </ul>
+   <div class="post_comment">
+   <% post.recentfivecomments.each do |comment| %>
+      <ul class="comment width"> 
+          <li> <%= comment.author.name %>: <%= comment.text%> </li>
+       </ul>
+       <% end %> 
+   </div>
+   <% end %>
+  </div>
+  <button type:submit class="all-posts">Pagination</buttom>
+</div>
```

Comment:
- [ ] kindly fix the image as it is too large
![Screenshot from 2023-05-20 07-15-20](https://github.com/Becky449/Myblog/assets/66097371/977c0777-93b8-4e78-b74d-85d1417ebdf1)